template-for-repository/proofs/Makefile-project-defines

```diff
@@ -13,6 +13,18 @@
 # Makefile.common.
 ################################################################
 
+# Name of this proof project, displayed in proof reports. For example,
+# "s2n" or "Amazon FreeRTOS". For projects with multiple proof roots,
+# this may be overridden on the command-line to Make, for example
+#
+# 	  make PROJECT_NAME="FreeRTOS MQTT" report
+#
+# PROJECT_NAME =
+
+# Path to litani executable, relative to the root of the repository.
+# This applies even for projects with multiple proof roots.
+# LITANI =
```

Comment:
I wonder whether I should modify the `setup.py` script to prompt the user for the Litani directory, and write it next to `SRCDIR`. The setup script could optionally add the submodule to the location user specified, if it didn't exist. Thoughts?

Reply:
It should prompt for those things that are required for make report to run.

Reply:
done, thanks